Dockerfile

```diff
@@ -39,7 +19,7 @@ COPY pkg/ $GOPATH/src/github.com/onosproject/simulators/pkg/
 RUN cd $GOPATH/src/github.com/onosproject/simulators && \
     GO111MODULE=on go get github.com/onosproject/simulators/cmd/gnmi_target
 
-FROM alpine:3.9
+FROM alpine:3.11
 RUN apk add --update bash openssl curl && rm -rf /var/cache/apk/*
 ENV ONOS_SIMULATORS_ROOT=$GOPATH/src/github.com/onosproject/simulators
```

Comment:
I think it should be gnxi-simulators

Reply:
no - because in the `go.mod` it's still "simulators"